src/scss/modules/sidebar.scss

```diff
@@ -38,6 +38,7 @@
 .sidebar__label {
   color: #{$black-60};
   display: inline-block;
+  flex-shrink: 0;
```

Comment:
This `.sidebar__label` is used across all the sidebars. So I doubled checked those and they don't appear to be effected by this.